socorro/processor/processor_app.py

```diff
@@ -109,17 +109,6 @@
         ),
     },
 
-    'processor': {
-        'processor_class': 'socorro.processor.processor_2015.Processor2015',
-
-        # These are for the minidump-stackwalker command line
-        'command_pathname': '/stackwalk/stackwalker',
-        'kill_timeout': 30,
-        'symbols_urls': ','.join([
-            'https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.symbols-public/v1',
-        ]),
-    },
-
```

Comment:
These behavior defaults overrode the existing defaults for local dev, but that's not what we want, yet. We want local dev values to be in `local_dev.env` and not scattered around. Everything else should default to -stage and -prod values.